Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -524,6 +521,12 @@ void CCodeView::OnPaint(wxPaintEvent& event)
 			{
 				desc = m_debugger->GetDescription(address);
 			}
+			
+			if (comment.empty())
```

Comment:
This is an accidental holdover from a point where comment.empty made sense, and I forgot to remove it. I don't see why not to include `if (!comment.empty())` though, so I'll just make it that.